<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNWListDialog</class>
 <widget class="QDialog" name="AddNWListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请输入新的单词本的名字：</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>设为默认单词本</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonSubmit">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddNWListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonSubmit</sender>
   <signal>clicked()</signal>
   <receiver>AddNWListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
